pkg/test/testharness/golden.go

```diff
@@ -11,10 +11,18 @@ import (
 )
 
 func RunGoldenTests(t *testing.T, basedir string, fn func(h *Harness, dir string)) {
-	files, err := ioutil.ReadDir(basedir)
+	entries, err := os.ReadDir(basedir)
 	if err != nil {
 		t.Fatalf("ReadDir(%q) failed: %v", basedir, err)
 	}
+	files := make([]fs.FileInfo, 0, len(entries))
```

Comment:
Not a blocker, but I hope we don't have to do this every time now with the switch away from ioutil :-)